gax-java/gax/src/main/java/com/google/api/gax/rpc/Watchdog.java

```diff
@@ -194,7 +194,7 @@ class WatchdogStream<ResponseT> extends StateCheckingResponseObserver<ResponseT>
     private volatile StreamController innerController;
 
     @GuardedBy("lock")
-    private State state = State.IDLE;
```

Comment:
Please add comment that by default that a stream has automatic inbound flow control enabled. So the stream will not wait for the caller to request() a message, so thats why this defaults to WAITING